js/modules/k6/grpc/client.go

```diff
@@ -117,7 +117,7 @@ func (c *Client) Connect(addr string, params map[string]interface{}) (bool, erro
 
 	p, err := c.parseConnectParams(params)
 	if err != nil {
-		return false, err
+		return false, fmt.Errorf("invalid grpc.Connect() parameters: %w", err)
```

Comment:
```suggestion
		return false, fmt.Errorf("invalid grpc.connect() parameters: %w", err)
```

The actual JS method name is with lowercase, see https://k6.io/docs/javascript-api/k6-net-grpc/client/client-connect